### PR TITLE
Clone from the ewasm-benchmarking organization

### PR DESCRIPTION
**Pull request by hugo-dc**
~~Depends on https://github.com/ewasm/benchmarking/pull/43~~
~~Also depends on Docker image `jwasinger/bench:1.0`.~~

Changes EVM Engine and Scout Engines Dockerfiles  to clone from the `ewasm-benchmarking` repositories.

### CODE REVIEW

**Comment by axic** [comment]
@hugo-dc can you pull out those changes to evm which don't depend on @jwasinger's PR and merge them now?

**Comment by hugo-dc** [comment]
Changed and merged according to @axic comments.

File: scout-engines/Dockerfile [hunk 1]
```diff
@@ -144,67 +144,42 @@ RUN cd wabt-bn128-rolluprs-fastmont-fasthost-superops && make clang-release
 #RUN cd wabt-bn128-rolluprs-fastmont-slowhost && make clang-release
 
 
-# wabt branch for wasmsnark bls12
-RUN git clone --recursive --single-branch --branch bls12-bignums-fastmont-superops https://github.com/ewasm/wabt.git wabt-bls12-fastmont-fasthost-superops
```
Comment (axic) [inline scout-engines/Dockerfile:148]: This seems to have dropped the additions from @cdetrio ?

Comment (cdetrio) [reply scout-engines/Dockerfile:148]: Yea... I'll rebase and add them.

**Pull request merged**